artiq/coredevice/zotino.py

```diff
@@ -0,0 +1,53 @@
+"""RTIO driver for the Zotino 32-channel, 16-bit 1MSPS DAC.
+
+Output event replacement is not supported and issuing commands at the same
+time is an error.
+"""
+
+from artiq.language.core import kernel
+from artiq.coredevice import spi2 as spi
+from artiq.coredevice.ad53xx import SPI_AD53XX_CONFIG, AD53xx
+
+_SPI_DAC_CONFIG = SPI_AD53XX_CONFIG
```

Comment:
I don't think this redefinition helps.

Reply:
Yes let's scrap it. Remnant f old version of code. Will fix.